misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -26,14 +28,32 @@ export default class ConfigComponent extends React.PureComponent<IConfigProps> {
 
   render() {
     const { resources } = this.props
-    return (
-      <div>
-        <H1>Config</H1>
-        {resources &&
-          Object.entries(resources).map(([name, file]) =>
-            this.renderConfig({ name, file })
-          )}
-      </div>
-    )
+    if (resources) {
+      return (
+        <div>
+          <H1>Config</H1>
+          {resources &&
+            Object.entries(resources).map(([name, file]) =>
+              this.renderConfig({ name, file })
+            )}
+        </div>
+      )
+    } else {
+      return (
+        <div>
+          <H1>Config</H1>
+          <br />
+          <H3 css={cssH3}>{"Effective Config"}</H3>
+          <CodePreContainer>{<Spinner />}</CodePreContainer>
+          <br />
+          <H3 css={cssH3}>{"classpath://common.yaml"}</H3>
```

Comment:
are these paths correct?

Reply:
This is the loading screen when data isn't present yet